### PR TITLE
feat(plugins): adds `projen` - A new generation of project generators

### PR DESCRIPTION
**Pull request by pgollucci**
Define and maintain complex project configuration through code.

JOIN THE #TemplatesAreEvil MOVEMENT!

projen synthesizes project configuration files such as package.json, tsconfig.json, .gitignore, GitHub Workflows, eslint, jest, etc from a well-typed definition written in JavaScript.

Check out this talk about projen.

As opposed to existing templating/scaffolding tools, projen is not a one-off generator. Synthesized files should never be manually edited (in fact, projen enforces that). To modify your project setup, users interact with rich strongly-typed class and execute projen to update their project configuration files.

### CODE REVIEW

**Review by eladb — APPROVED** [review]

**Comment by pgollucci** [comment]
ping?

File: plugins/projen/README.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+```zsh
+plugins=(... projen)
+```
+
+## Plugin commands
+
+* `pj_install` installs the `npm` module globally
+
+* `pgjn='projen new'`
+* `pgjv='projen --version'`
```
Comment (eladb) [inline plugins/projen/README.md:16]: Shouldn't these be `pjn` and `pjv`?

Comment (pgollucci) [reply plugins/projen/README.md:20]: pj = local via npx projen
pg = global (from npm -g)

so pgjv is the global projen version
and pjv is the local projen version

Comment (pgollucci) [reply plugins/projen/README.md:20]: That said, I'm happy to change any of them

**Comment by TheTechRobo** [comment]
Why is  it closed?

**Pull request closed without merging**